tests/Fixtures/TestBundle/Entity/ThirdLevel.php

```diff
@@ -51,6 +53,14 @@ class ThirdLevel
     #[ORM\ManyToOne(targetEntity: FourthLevel::class, cascade: ['persist'])]
     public $badFourthLevel;
 
+    #[ORM\OneToMany(mappedBy: 'thirdLevel', targetEntity: RelatedDummy::class)]
```

Comment:
if you change this test (and the `RelateDummy`) you need to also add the change to the Document of the same names for mongodb. Note that we recommend not changing existing features to avoid this (I'm fine with replicating the change to documents).

Reply:
Thanks for the tip, I had definitely missed this part about MongoDB in the contributing file. Should be fine now.